src/screen/dashboard.rs

```diff
@@ -2008,18 +2057,27 @@ impl Dashboard {
     ) -> (Self, Task<Message>) {
         use pane_grid::Configuration;
 
-        fn configuration(pane: data::Pane) -> Configuration<Pane> {
+        fn configuration(
+            pane: data::Pane,
+            buffer_settings: &dashboard::BufferSettings,
+            config: &Config,
+        ) -> Configuration<Pane> {
             match pane {
                 data::Pane::Split { axis, ratio, a, b } => Configuration::Split {
                     axis: match axis {
                         data::pane::Axis::Horizontal => pane_grid::Axis::Horizontal,
                         data::pane::Axis::Vertical => pane_grid::Axis::Vertical,
                     },
                     ratio,
-                    a: Box::new(configuration(*a)),
-                    b: Box::new(configuration(*b)),
+                    a: Box::new(configuration(*a, buffer_settings, config)),
+                    b: Box::new(configuration(*b, buffer_settings, config)),
                 },
-                data::Pane::Buffer { buffer, settings } => {
+                data::Pane::Buffer { buffer, .. } => {
+                    let settings = buffer_settings
+                        .get(&buffer)
+                        .cloned()
+                        .unwrap_or(config.buffer.clone().into());
+
                     Configuration::Pane(Pane::with_settings(Buffer::from(buffer), settings))
```

Comment:
We don't even need settings at the pane level anymore. We're duplicating it in two places. We can just pass buffer_settings down during view and update for the buffers to use instead of storing it there as well